src/transformers/__init__.py

```diff
@@ -7831,6 +7841,15 @@
             VivitModel,
             VivitPreTrainedModel,
         )
+
+        # PyTorch model imports
+        from .models.vmamba import (
+            VMAMBA_PRETRAINED_MODEL_ARCHIVE_LIST,
+            VMambaForImageClassification,
+            VMambaModel,
+            VMambaPreTrainedModel,
+            load_tf_weights_in_vmamba,
```

Comment:
This shouldn't be imported here. I think this is the reason for the TYPE_HINT errors